mineral-ui/mineral-ui
# [Button]: filter the `element` prop from being passed down

### Description

Remove the element prop from the props being passed to the DOM when Button will be rendered as an 'a' tag.

### Motivation and context

closes #659 

### Screenshots, videos, or demo, if appropriate

https://659-button-element-props--mineral-ui.netlify.com/

### How to test


1. `cd mineral-ui && npm start`.
2. Navigate to [Button > Elements](http://0.0.0.0:8080/components/button#primary) section.
3. Check your console that there are no errors/warnings.
4. Make sure the rest of the demos look laid out correctly.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)


### Checklist




### How does this PR make you feel?

![huh](https://media.giphy.com/media/8GbvxxFHJDBa8/giphy.gif)
